packages/zero-server/package.json

```diff
@@ -23,6 +23,8 @@
   "devDependencies": {
     "@rocicorp/prettier-config": "^0.3.0",
     "@vitest/runner": "3.2.4",
+    "drizzle-orm": "^0.44.0",
+    "postgres": "^3.4.0",
```

Comment:
Do we also want peer dependencies? And make sure that these are part of the `@rocicorp/zero` npm package too.